templates/helm/values.yaml.tpl

```diff
@@ -47,6 +47,10 @@ log:
 # If specified, the service controller will watch for object creation only in the provided namespace
 watchNamespace: ""
 
+# If specified, the service controller will use Role/Rolebinding instead of ClusterRole/ClusterRoleBindings
```

Comment:
nit: Make the description a little higher level. Suggestion like "If specified, the service controller will only be able to operate within a single namespace"